src/opm/input/eclipse/Schedule/Network/ExtNetwork.cpp

```diff
@@ -113,15 +142,15 @@ std::optional<Branch> ExtNetwork::uptree_branch(const std::string& node) const {
     if (branches.size() == 1)
         return std::move(branches[0]);
 
-    throw std::logic_error("Bug - more than upstree branch for node: " + node);
+    throw std::logic_error("Bug - more than one uptree branch for node: " + node);
 }
 
 
 std::vector<Branch> ExtNetwork::downtree_branches(const std::string& node) const {
-    if (!this->has_node(node))
```

Comment:
I think we should keep the `throw` until there is a need to deal with nodes that are not part of the network.

Reply:
Ok!